lib/suspenders/app_builder.rb

```diff
@@ -98,7 +102,9 @@ def enable_rack_canonical_host
       config = <<-RUBY
 
   # Ensure requests are only served from one, canonical host name
-  config.middleware.use Rack::CanonicalHost, ENV.fetch("APPLICATION_HOST")
+  unless ENV["HEROKU_APP_NAME"].include?("staging-pr-")
```

Comment:
Because we just don't want this to run on review apps, which will have the `pr` part. We want this on regular staging.